code/_onclick/click.dm

```diff
@@ -380,6 +380,10 @@ GLOBAL_LIST_INIT(click_catchers, create_click_catcher())
 /obj/screen/click_catcher/Destroy()
 	return QDEL_HINT_LETMELIVE
 
+/obj/screen/click_catcher/New()
```

Comment:
I am pretty sure you need to keep arguments when making a subtype of a proc.

Reply:
there is ..()?

Reply:
> there is ..()?

/obj/screen/click_catcher/New(_name = "", mob/living/_parentmob, _icon, _icon_state)